include/multipass/virtual_machine_factory.h

```diff
@@ -61,6 +62,12 @@ class VirtualMachineFactory
                                                   const Path& cache_dir_path, const Path& data_dir_path,
                                                   const days& days_to_expire) = 0;
 
+    // List all the network interfaces seen by the backend.
+    virtual std::vector<NetworkInterfaceInfo> list_networks() const = 0;
+
+    // Given the id the user sees in list_networks(), return the id of an interface which identifies it.
+    virtual std::string low_level_id(const std::string& ux_id) const = 0;
```

Comment:
Looking again, I don't think the function name is clear in this context. Low level id of what? I know it's explained in the comment, but the name is the first form of documentation (it's what's visible at the calling site). Perhaps `reinterpret_interface_id`?

Reply:
Makes sense. I edited the old commit which introduced this, so there are not traces of `low_level_id` anymore. This will be reflected when I force-push.